src/Result.php

```diff
@@ -282,6 +282,36 @@ public function columnCount()
 
 
     /**
+     * Get the iterator data as an array.
+     *
+     * @return array
+     */
+    public function getDataAsArray()
+    {
+        return iterator_to_array($this->result);
```

Comment:
I did this so it could be used in other cases, but I didn't see the getValues method, so this method is redundant. I shall remove.